config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   root 'users#show'
   get '/spa', to: 'spa#index'
+  resources :entries, defaults: { format: 'json' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.